pkg/sql/sem/tree/import.go

```diff
@@ -16,7 +16,7 @@ package tree
 
 // Import represents a IMPORT statement.
 type Import struct {
-	Table      UnresolvedName
+	Table      NormalizableTableName
```

Comment:
I'm not completely sure what effect this has on IMPORT, but ISTM you need a new test to prove it one way or another, right?